src/qt/window.cpp

```diff
@@ -1490,7 +1490,10 @@ bool wxWindowQt::QtHandleCloseEvent ( QWidget *handler, QCloseEvent *WXUNUSED( e
     if ( GetHandle() != handler )
         return false;
 
-    return Close();
+    if (!IsEnabled())
+        return true;
```

Comment:
Why should the window care if it was enabled or not? `Close()` can be called programmatically for a disabled window as well. Should it not close in that case?

Reply:
This was to restore the behaviour that was previously present in the MSW version, where clicking the close button of a disabled window would not close the window or generate a wx event.

Reply:
You mean that right now, a disabled top level window will still react to a click on the [also disabled] close button?!

Reply:
Yes, I verified this in the 'dialogs' sample by opening: Dialogs -> Informative Dialogs -> Busy Info Dialog and then clicking the close button on the parent window. I guess the problem is more that Qt generates close events even when the window is disabled but I could not find any way to change this behaviour, other than dealing with the qt close event.

Reply:
Amazing... oh well..
Can you please add a comment explaining this, maybe before the `if`?

Also, please correct the spaces for the same `if (  )`.